### PR TITLE
Remove Airlock component from builder

### PR DESCRIPTION
**Pull request by chefsalim**
This is the last part of the Airlock transition - this removes the Airlock component from builder repo, and also removes it as a dep for builder-worker, along with some associated tweaks.

Signed-off-by: Salim Alam <salam@chef.io>

tenor-170761497

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ env:
   global:
     - PATH=$HOME/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin:$HOME/.cargo/bin:$PATH
     # Habitat Rust program components
-    - _RUST_HAB_BIN_COMPONENTS="components/airlock"
+    - _RUST_HAB_BIN_COMPONENTS=""
```
Comment (raskchanky) [inline .travis.yml]: Is this worth keeping at all if it's empty?

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 UNAME_S := $(shell uname -s)
-BIN = airlock
+BIN =
```
Comment (raskchanky) [inline Makefile]: Same question here as above.  Why not just delete the whole thing?

File: .travis.yml (hunk 1 above)
Comment (chefsalim) [reply .travis.yml]: Yeah I started to remove all the bin plumbing  but we may have a new bin tool in the (near to mid term) future so leaving it in for now..

**Pull request merged**